search.php

```diff
@@ -12,7 +12,7 @@
 /**
  * Planet4 - Search functionality.
  */
-
+wp_enqueue_script( 'search', get_template_directory_uri() . '/assets/js/search.js', [], '0.1.0', true );
```

Comment:
@kirdia could you place this inside the if statement and at the end of it?

Reply:
Done :+1: